.github/workflows/publish.yaml

```diff
@@ -0,0 +1,51 @@
+jobs:
+  release_pypi:
+      name: Build and publish Python distributions 📦 to PyPI and TestPyPI
+      runs-on: ubuntu-latest
+      if: github.event_name == 'push' && startsWith(github.ref, 'refs/tags')
+      steps:
+        - uses: actions/checkout@v3
+        - name: Set up Python 3.10
+          uses: actions/setup-python@v4
+          with:
+            python-version: "3.10"
+            cache: 'pip'
+        - name: Install pypa/build
+          run: |
+            pip install --upgrade build
+        - name: Build a binary wheel and a source tarball
+          run: |
+            python -m build --sdist --wheel --outdir dist/
+        - name: Publish distribution 📦 to Test PyPI
```

Comment:
Does this need to be published on Test PyPi and PyPi?

Reply:
Do you mean do we need to push it to PyPI or push it to both?

Reply:
I'm just saying "Publish distribution 📦 to Test PyPI"  and "Publish distribution 📦 to PyPI" sections are both included and I'm just asking if it was intended to include both? Is it being pushed to Test PyPI first in case this falls over?

Reply:
Oh yeah, that's the workflow recommended in the [Python packaging tutorial for GHA](https://packaging.python.org/en/latest/guides/publishing-package-distribution-releases-using-github-actions-ci-cd-workflows/).

Reply:
I think it's to make sure bad packages don't get pushed to PyPI.